benchkit/devices/adb/__init__.py

```diff
@@ -43,23 +45,40 @@ def is_connected(self) -> bool:
         return "device" == self.status
 
 
-class AndroidDebugBridge:  # TODO add commlayer for "host"
+# TODO: investigate the identifier and daemon. temporarily it's mirrored like HDC does it.
```

Comment:
have you tested your solution with Android?

Reply:
partially, I will add a test to demonstrate and test this behavior.